### DISCUSSION TITLE
Vercel project should rebuild on ENV change

### DISCUSSION THREAD

**Discussion by ivan-kleshnin** [vercel/vercel#4480] [Ideas] [answered]
Vercel deployments should rebuild / redeploy if any of the environment variables were changed.
This is a standard behavior on Heroku and I think it makes perfect sense.

Currently I have to add *noop* commits or fallback to `$ now` (with its quotas) just to rebuild/redeploy a project with new env. variables.

**Comment by leo** [comment, marked as answer]
Thank you for sharing!

This is definitely something we're considering for the near future. We're likely gonna have a UI that lets you easily issue a new build right from the fieldset that you've changed.
